distribution.pm

```diff
@@ -130,15 +130,16 @@ sub script_run {
     my ($self, $cmd) = splice(@_, 0, 2);
     my %args = testapi::compat_args(
         {
-            timeout => $bmwqemu::default_timeout,
-            output  => '',
-            quiet   => undef
+            timeout    => $bmwqemu::default_timeout,
+            output     => '',
+            quiet      => undef,
+            type_speed => $bmwqemu::vars{VNC_TYPING_LIMIT}
```

Comment:
- The default for **max_interval** is 250: https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L1408
- The default for **VNC_TYPING_LIMIT_DEFAULT** is 30: https://github.com/os-autoinst/os-autoinst/blob/master/consoles/vnc_base.pm#L33
- [Doc](https://github.com/os-autoinst/os-autoinst/blob/master/doc/backend_vars.asciidoc) is confusing. Default of **VNC_TYPING_LIMIT** is 50?

In any case, this looks wrong. It is going to cause a slower typing than before each time `script_run` is used.

Reply:
Should I hardcode it then to accept e.g. 'normal', 'slow' and 'fast'?

Reply:
```perl
type_speed => undef
```

Keep in mind that this change is for distribution, not only VNC based installations.
The function type_string for VNC is on: https://github.com/os-autoinst/os-autoinst/blob/master/consoles/vnc_base.pm#L98
And it already make use of
```perl
my $seconds_per_keypress = 1 / (get_var('VNC_TYPING_LIMIT', VNC_TYPING_LIMIT_DEFAULT) || 1);
```